includes/uninstall.php

```diff
@@ -0,0 +1,134 @@
+<?php
+/**
+ * Uninstall helpers.
+ *
+ * @package   Google\Web_Stories
+ * @copyright 2020 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://github.com/google/web-stories-wp
+ */
+
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Web_Stories;
+
+/**
+ * Deletes options and transients.
+ *
+ * @return void
+ */
+function delete_options() {
+	global $wpdb;
+
+	$prefix = 'web_stories\_%';
+
+	// phpcs:ignore WordPress.DB.DirectDatabaseQuery
+	$options = $wpdb->get_results(
+		$wpdb->prepare(
+			"SELECT option_name FROM $wpdb->options WHERE option_name LIKE %s",
+			$prefix
+		),
+		ARRAY_N
+	);
+
+	if ( ! empty( $options ) ) {
+		array_map( 'delete_option', (array) $options );
+	}
+
+
+	// phpcs:ignore WordPress.DB.DirectDatabaseQuery
+	$transients = $wpdb->get_results(
+		$wpdb->prepare(
+			"SELECT option_name FROM $wpdb->options WHERE option_name LIKE %s OR option_name LIKE %s",
+			'_transient_' . $prefix,
+			'_transient_timeout_' . $prefix
+		),
+		ARRAY_N
+	);
+
+	if ( ! empty( $transients ) ) {
+		array_map( 'delete_transient', (array) $transients );
```

Comment:
```suggestion
		array_map( 'delete_option'', (array) $transients );
```

Reply:
Interesting, can you elaborate? I split up the queries & used `delete_transient` on  purpose due to different actions and cache groups.

Reply:
If you look the code in `delete_transient` it adds prefix to the option name. Either we strip this out or just use `delete_option`